schema/test/unit/tables/revision_status_test.sql

```diff
@@ -74,7 +75,20 @@ select is (
   'sets is_amendment_specific to false when status is Discussion / Pending Province Approval / Pending Proponent Signature'
 );
 
-
+select results_eq(
+  $$
+    select name, sorting_order from cif.revision_status order by sorting_order
+  $$,
+  $$
+    values
+      ('Draft'::varchar, 0::int),
+      ('In Discussion'::varchar, 0::int),
```

Comment:
Is there a reason for that? I think it only matters to have a sequence for displaying statuses unless there's a reason I'm not aware of.

Reply:
I don't know if there's a reason. My best guess would be because [postgres indexing starts at 1](https://www.postgresql.org/docs/current/arrays.html#:~:text=The%20array%20subscript%20numbers%20are%20written%20within%20square%20brackets.%20By%20default%20PostgreSQL%20uses%20a%20one%2Dbased%20numbering%20convention%20for%20arrays%2C%20that%20is%2C%20an%20array%20of%20n%20elements%20starts%20with%20array%5B1%5D%20and%20ends%20with%20array%5Bn%5D.)